client/src/components/Public/EntitySearch/EntitySearch.js

```diff
@@ -28,6 +28,21 @@ import {updateValue} from '../../../actions/sharedActions'
 import {FIND_ENTITY_TITLE} from '../../../constants'
 import './EntitySearch.css'
 
+type EntitySearchProps = {
```

Comment:
Use exact types, when possible `{|`. This way if you pass some unused prop, flow will tell you.